src/pages/Organization/OrganizationPatients.tsx

```
@@ -28,10 +28,39 @@ interface Props {
+    };
+    updateQuery(searchParams);
+  }, []);
+
+  const handleFieldChange = () => {
+    {
+      updateQuery({
+        name: undefined,
+        phone_number: undefined,
+      });
+      clearSearch.value = true;
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)